<launch>
  <node name="multiagent_node_10" pkg="multiagent" type="multiagent_node" output="screen" >
    <param name="test_id" value="1_0"/>
    <param name="total_robots" value="2" />
    <param name="allocated_time" value="0.1" />
    <param name="true_map_topic" value="/truemap" />
    <param name="common_map_topic" value="/commonmap" />
    <param name="base_folder" value="$(find multiagent)" />
    <param name="output_folder" value="/home/test_results" />
    <param name="communication_epsilon" value="1.5"/>
  </node>
</launch>
